src/shapes/group.class.js

```diff
@@ -410,7 +404,6 @@
      * @param {CanvasRenderingContext2D} ctx Context to render on
      */
     drawObject: function(ctx) {
-      this._renderBackground(ctx);
```

Comment:
any reason why you let this go at the end?

Reply:
https://github.com/fabricjs/fabric.js/pull/7859#discussion_r846930838

Reply:
solved by #7901